frontend/src/containers/Homepage/Homepage.js

```diff
@@ -31,6 +33,12 @@ const Homepage = (props) => {
               <h5>Based on this week&apos;s popular ingredient donations</h5>
             </div>
           </div>
+          <div className="row">
+            <SearchResults searchTerm={searchTerm} />
```

Comment:
Maybe we should call this something like `ResultCounter`? `SearchResults` is a little ambiguous.